client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/step-content.tsx

```diff
@@ -58,25 +59,36 @@ const StepContent = ( {
 	// The adminUrl points to the .wordpress.com domain for this site, so we'll use this for the preview.
 	const iFrameURL = adminUrl ? new URL( adminUrl as string ).host : null;
 
+	const header = (
+		<>
+			<WordPressLogo className="launchpad__sidebar-header-logo" size={ 24 } />
+			<span className="launchpad__sidebar-header-flow-name">{ flowName }</span>
+		</>
+	);
+
+	const sidebar = (
+		<Sidebar
+			sidebarDomain={ sidebarDomain }
+			launchpadKey={ launchpadKey }
+			siteSlug={ siteSlug }
+			submit={ submit }
+			goNext={ goNext }
+			goToStep={ goToStep }
+			flow={ flow }
+		/>
+	);
+
 	return (
-		<main className="launchpad__container">
-			<div className="launchpad__sidebar-header">
-				<WordPressLogo className="launchpad__sidebar-header-logo" size={ 24 } />
-				<span className="launchpad__sidebar-header-flow-name">{ flowName }</span>
-			</div>
-			<div className="launchpad__content">
-				<Sidebar
-					sidebarDomain={ sidebarDomain }
-					launchpadKey={ launchpadKey }
-					siteSlug={ siteSlug }
-					submit={ submit }
-					goNext={ goNext }
-					goToStep={ goToStep }
-					flow={ flow }
-				/>
-				<LaunchpadSitePreview flow={ flow } siteSlug={ iFrameURL } />
-			</div>
-		</main>
+		<LaunchpadContainer
+			headerClassName="launchpad__sidebar-header"
```

Comment:
In the future, if we could kill all of these classes and the overrides, keeping only the new ones from the `StepContainer`, it would be ideal.

But I don't feel confident enough to do it without breaking things that I wouldn't think about.